app/assets/admin/css/admin.css

```diff
@@ -159,8 +159,8 @@
   -webkit-box-flex: 0;
       -ms-flex: 0 0 auto;
           flex: 0 0 auto;
-  padding-right: 0.5rem;
-  padding-left: 0.5rem;
+  padding-right: .5rem;
```

Comment:
please do not remove the leading 0. fix on the other lines too where you removed.

Reply:
Grunt build files are .min.css . please check again if its converted by grunt.